app/controllers/worker_profiles_controller.rb

```diff
@@ -13,16 +13,26 @@ def new
   end
 
   def create
-    @worker_profile = WorkerProfile.create!(worker_params)
+    @worker_profile = WorkerProfile.new(worker_params)
```

Comment:
why do we need 2 variables? either we use create or new whould be enough right?

Reply:
we dont need two varaible the red line is the deleted line and the green line is the new one. Basically the green is replacing the red line, its the difference between old file and the new one.